.vsts-ci/windows-daily.yml

```diff
@@ -0,0 +1,83 @@
+name: PR-$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:.rr)
+trigger:
+  # Batch merge builds together while a merge build is running
+  batch: true
+  branches:
+    include:
+    - master
+    - release*
+  paths:
+    include:
+    - '*'
+    exclude:
+    - /tools/releaseBuild/**/*
+    - /.vsts-ci/misc-analysis.yml
+    - /.github/ISSUE_TEMPLATE/*
+pr:
+  branches:
+    include:
+    - master
+    - release*
+  paths:
+    include:
+    - '*'
+    exclude:
+    - /tools/releaseBuild/**/*
+    - /.vsts-ci/misc-analysis.yml
+    - /.github/ISSUE_TEMPLATE/*
+
+variables:
+  GIT_CONFIG_PARAMETERS: "'core.autocrlf=false'"
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+  POWERSHELL_TELEMETRY_OPTOUT: 1
+  # Avoid expensive initialization of dotnet cli, see: https://donovanbrown.com/post/Stop-wasting-time-during-NET-Core-builds
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+
+resources:
+- repo: self
+  clean: true
+
+jobs:
+- template: templates/ci-build.yml
+
+- template: templates/windows-test.yml
+  parameters:
+    purpose: UnelevatedPesterTests
+    tagSet: CI
+    parentJobs:
+      - win_build
+
+- template: templates/windows-test.yml
+  parameters:
+    purpose: ElevatedPesterTests
+    tagSet: CI
+    parentJobs:
+      - win_build
+
+- template: templates/windows-test.yml
+  parameters:
+    purpose: UnelevatedPesterTests
+    tagSet: Others
+    parentJobs:
+      - win_build
+
+- template: templates/windows-test.yml
+  parameters:
+    purpose: ElevatedPesterTests
+    tagSet: Others
+    parentJobs:
+      - win_build
+
+- template: templates/windows-packaging.yml
```

Comment:
Add a comment why we wait here and in the CI yml why we don't

Reply:
Fixed.